crates/sui-framework/sources/governance/sui_system.move

```diff
@@ -80,6 +80,13 @@ module sui::sui_system {
         epoch_start_timestamp_ms: u64,
     }
 
+    struct SuiSystemStateWrapper has key {
+        id: UID,
+        version: u64,
+        // TODO: Make it a dynamic object field
```

Comment:
Any reason to make it a dynamic *object* field (vs just a dynamic field) -- i.e. will we need to know the ID of the inner object off-chain?  Asking because there is an ongoing conversation about getting rid of dynamic object field, in favour of indexing wrapped objects.

Reply:
cc @tnowacki

Reply:
We don't need to know the ID of the inner object off-chain, as long as there is an easy way to access this child object in Rust

Reply:
I think you will, in any case, need to access the inner object via its parent, so doing it via a dynamic field will save you one step.